### DISCUSSION TITLE
Pokedex

### DISCUSSION THREAD

**Discussion by ghost** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#419] [Ideas]
Buenas noches compañeros, una pregunta alguien sabe como pasar el texto de la api como lo podría ser el tipo de pokemon al HTML

**Comment by DanyVeneno** [comment]
Hola Nathanael no es muy puntual tu pregunta pero te dejo un ejemplo de como podria ser y te dejo este link para que analizes el codigo y te orientes masomenos . https://codepen.io/kyla-burgess/pen/YzyWwrw
saludos buena noche

**Reply by ghost** [reply]
Buenas noches Daniel, muchas gracias por la información, me será de gran ayuda!!

**Comment by Jose-Galicia** [comment]
![image](https://user-images.githubusercontent.com/99226240/158721644-b016700d-160d-408c-9838-0c01ba43d5d8.png)
Hola, aun estoy trabajando en eso, pero de esta manera puedes pasar el texto de la api a el html.

**Reply by ghost** [reply]
Muchas gracias jose!!

**Comment by DanyVeneno** [comment]
Gracias José
